packages/zipkin/src/InetAddress.js

```diff
@@ -18,16 +18,4 @@ class InetAddress {
   }
 }
 
-// In non-node environments we fallback to 127.0.0.1
-InetAddress.getLocalAddress = function getLocalAddress() {
-  const isNode = typeof process === 'object' && typeof process.on === 'function';
-  if (!isNode) {
-    return new InetAddress('127.0.0.1');
-  }
-
-  // eslint-disable-next-line global-require
-  const networkAddress = require('network-address');
```

Comment:
if you do `require.apply(null, 'network-address')` you fool the react native bundler into not bundling this. as long as `isNode === false` above, that should fix the problem

Reply:
That should negate the need for an API change

Reply:
Wow, thanks for the hints @SimenB, will try that!